src/components/Table/GridTable.test.tsx

```diff
@@ -1592,6 +1593,29 @@ describe("GridTable", () => {
     // But the 2nd added row did not
     expect(row(r, 2).getAttribute("data-render")).toEqual("1");
   });
+
+  it("reacts to setting activeRowId", async () => {
+    const activeRowIdRowStyles: GridRowStyles<Row> = {
+      data: {
+        onClick: (row, api) => {
+          api.setActiveRowId(`${row.kind}`);
```

Comment:
Kinda expect to see row.id in here?

Reply:
oh yea.. i wonder why this worked....

Reply:
Ahh, because the second color is the same as the first... whoops!

Reply:
Okay, all fixed.